### ISSUE TITLE
Implement group change

### ISSUE THREAD

**Issue by lkuchenb** [ghga-de/datameta#22]
Group change requires checking for data that is associated with the old user/group combination.

**Comment by KerstenBreuer** [comment]
All data uploaded by users of a specific group should be own by the group not my the user right?
Then we will have no problems if a user changes groups, right?

@lkuchenb

**Comment by lkuchenb** [comment]
Yes. This ticket is primarily about the pending submissions, since this is currently owned by the UID/GID combination (such that different users of the same lab can work on their submissions without crosstalk). So I guess the solution to this ticket is simply to clean up all pending submissions of the user when he changes group, I just wanted to keep in mind that this has to be taken care of so I wrote it down here. To avoid pending submission zombies that are owned by UID/GID tuples that don't exist anymore.

**Comment by KerstenBreuer** [comment]
OK, right that makes sense.
